packages/firebase_ml_vision/example/android/app/build.gradle

```diff
@@ -31,9 +31,13 @@ android {
         disable 'InvalidPackage'
     }
 
+    aaptOptions {
+        noCompress "tflite"
+    }
+
     defaultConfig {
         applicationId "io.flutter.plugins.firebasemlvisionexample"
-        minSdkVersion 16
+        minSdkVersion 21
```

Comment:
Does Auto ML Vision Edge require 21+ or is this for adding the `camera` plugin?

Reply:
that was for the camera plugin - will revert!